app/scripts/models/account.js

```diff
@@ -26,6 +26,7 @@ define([
     grantedPermissions: undefined,
     lastLogin: undefined,
     needsOptedInToMarketingEmail: undefined,
+    hasHadProfileImage: undefined,
```

Comment:
could we update this to 
`hadProfileImage` or `hadProfileImageSet` or `hadProfileImageSet`?